screen-search/src/main/java/com/ivy/search/SearchViewModel.kt

```diff
@@ -29,48 +24,52 @@ class SearchViewModel @Inject constructor(
     private val categoriesAct: CategoriesAct,
     private val baseCurrencyAct: BaseCurrencyAct,
     private val allTrnsAct: AllTrnsAct
-) : ViewModel() {
+) : ComposeViewModel<SearchState, SearchEvent>() {
 
-    private val _baseCurrencyCode = MutableStateFlow(getDefaultFIATCurrency().currencyCode)
-    val baseCurrencyCode = _baseCurrencyCode.asStateFlow()
+    private val _uiState = mutableStateOf(
```

Comment:
Remove the `_` from the private vala name. There's no longer a need for underscore.